File: c - projects/factorial.c
#include<stdio.h>
int main(){
	int i=1,n,ans=1;
	printf("Enter a number - ");
	scanf("%d",&n);
	while(i<=n){
		ans=ans*i;
		i++;
	}
	printf("Ans = %d",ans);
	return 0;
}

File: dart/LAB-3/factorial.dart
import 'dart:io';

void main() {
  stdout.write("Enter number : ");
  stdout.writeln("${factorial(int.parse(stdin.readLineSync()!))}");
}

int factorial(int n) {
  if (n <= 0) {
    return 1;
  }
  return n * factorial(n - 1);
}
